ui/src/main.rs

```diff
@@ -136,17 +132,33 @@ pub enum AuthPages {
     Success,
 }
 
+#[derive(clap::Subcommand, Debug)]
+enum LogProfile {
+    /// normal operation
+    Normal,
+    /// print everything but tracing logs to the terminal
+    Debug,
+    /// print everything including tracing logs to the terminal
+    Trace,
+}
+
 #[derive(Debug, Parser)]
 #[clap(name = "")]
 struct Args {
+    /// The location to store the .uplink directory, within which a .warp, state.json, and other useful logs will be located
     #[clap(long)]
     path: Option<PathBuf>,
     #[clap(long)]
     experimental_node: bool,
     // todo: when the app is mature, default mock to false
     // there's no way to set --flag=true so for make the flag mean false
+    /// mock data is fake friends, conversations, and messages, which allow for testing the UI.
+    /// may cause crashes when attempting to add/remove fake friends, send messages to them, etc.
```

Comment:
Though will be a separate issue, but I feel the mock data should be pushed behind a feature gate in the future instead of a flag, especially when it comes close to production so nobody could easily use this flag can cause conflicts.